.github/workflows/cspell.yml

```diff
@@ -0,0 +1,24 @@
+name: CSpell
+on: push
+jobs:
+  cspell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14'
+      - name: Determine Node Version
+        id: node-version
+        run: |
+          echo "::set-output name=ver::$(node --version)"
+      - uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-${{ steps.node-version.outputs.ver }}-${{ hashFiles('package.json', 'yarn.lock') }}
```

Comment:
If we switch to `@v2`+ and use it's built-in cache you could remove these steps.

Reply:
unnecessary steps removed!